Modules/@babylonjs/react-native/windows/scripts/Setup.ps1

```diff
@@ -1,5 +1,18 @@
 Import-Module $PSScriptRoot\Utils.psm1

```

Comment:
What is the advantage of having these separate PowerShell scripts as opposed to just having all this logic in `gulpfile.js`?

Reply:
The only thing here currently that benefits from powershell scripts is the logic for resolving msbuild. For local developer setups where msbuild isn't added to the path variable, we download the vs installer powershell module which helps us resolve the directory to look in for msbuild. I'm not sure if something similar exists with gulp.

Reply:
I am going to keep some of the helper logic in the Utils.psm1 script, but i have reduced powershell usage. Setup is now conducted through gulp tasks. Only build logic is facilitated through powershell now.